### ISSUE TITLE
Footer does not respect navigation length in legacy docs

### ISSUE THREAD

**Issue by ChrisMaddock** [nunit/nunit.github.io#32]
![image](https://user-images.githubusercontent.com/4837132/29892568-f5f8dc74-8dc6-11e7-9683-da6371ef6315.png)

See http://nunit.org/docs/2.5.1/attributes.html. Seems to occur when the nav bar is taller than the content on the page.

**Comment by halex2005** [comment]
It is because of absolute positioning of sidenav. We need to change markup of legacy docs. Can we do so?

**Comment by halex2005** [comment]
@rprouse, I think it is possible to automatically extract text from old docs to markdown and then generate html using jekyll. Then we can use one markup for all the docs. What do you think?

**Comment by rprouse** [comment]
Doing so may break the multi language code blocks. An interesting experiment might be to grab the original doc fragments from the old site's repo and use JavaScript to reconstruct them on the fly. Originally it was done with PHP. That would reduce the size and compile time of the site considerably.

**Comment by ChrisMaddock** [comment]
@SeanKilleen - I've just been closing out some of my old issues, should this now be in the docs repo?

**Comment by SeanKilleen** [comment]
@ChrisMaddock yep, this can be moved to an issue in the docs repo and closed out here if you'd like 👍
